### PR TITLE
feature: adding intial set up for cypress and e2e test case for the about page

### LINKED ISSUE #26: Chore -> Add Cypress Automation Test Framework and update docs

### PR DESCRIPTION
**Pull request by saurabhsubham113**
About page

# Description
Added initial setup for the cypress automation and an e2e test case for the website's **about** page.

Fixes #26 

**Snapshots/Videos:**
Screenshot (1)
Screenshot (2)
Screenshot (3)
Screenshot (4)

## Type of change

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [x] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert
- [ ] ✏️ Updated Test Cases

## Added tests?

- [x] 👍 Yes
- [ ] 🙅 No, because they aren't needed
- [ ] 🙋 No, because I need help

## Internationalization Support?

- [ ] 👍 Yes, Added text tokens
- [x] 🙅 No, because they aren't needed
- [ ] 🙋 No, because I need help

**Steps to test the feature:**

**Snapshot of the test-cases that are passing:**

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

### CODE REVIEW

File: cypress/support/e2e.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
```
Comment (Neha) [inline cypress/support/e2e.ts:19]: please remove this commented code

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Neha) [inline cypress/fixtures/example.json:1]: Are we using these? If not, can we remove it. 
If yes, can we keep the name of the file more meaningful

File: cypress.config.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
+    baseUrl:"http://localhost:3000",
```
Comment (Neha) [inline cypress.config.ts:5]: Please create a config file and move this URL to config file and use the variable

File: cypress/e2e/about.cy.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
+describe("template spec", () => {
+    it("about section renders correctly", () => {
```
Comment (Neha) [inline cypress/e2e/about.cy.ts:2]: it is section or page?

**Review by Neha — CHANGES_REQUESTED** [review]

File: cypress/support/e2e.ts (hunk 1 above)
Comment (saurabhsubham113) [reply cypress/support/e2e.ts:19]: sure.
i'll remove out the comments

File: cypress/fixtures/example.json (hunk 2 above)
Comment (saurabhsubham113) [reply cypress/fixtures/example.json:1]: no this is just a sample file.
i'll remove this too.

File: cypress/e2e/about.cy.ts (hunk 4 above)
Comment (saurabhsubham113) [reply cypress/e2e/about.cy.ts:2]: this is the about page.

**Comment by saurabhsubham113** [comment]
Hi @Neha ,

I have resolved the PR comments. Please review and any feedback is welcomed.

**Comment by anandaryan008** [comment]
Hi @Neha 

can you please let me know if this PR has been reviewed or if you have any other PR comment

**Review by Neha — APPROVED** [review]
thank you for your PR.

**Pull request merged**